crates/pop-parachains/src/new_parachain.rs

```diff
@@ -19,22 +22,22 @@ use walkdir::WalkDir;
 /// * `tag_version` - version to use (`None` to use latest).
 /// * `config` - customization values to include in the new parachain.
 pub fn instantiate_template_dir(
-	template: &Template,
+	template: &ParachainTemplate,
```

Comment:
Do we need a borrow here?

Reply:
If a template was just a regular enum, I would say not. However, because templates have a lot of associated information via Strum, I imagine a copy would not be ideal.